lib/ui/fixtures/ui_test.dart

```diff
@@ -747,9 +747,50 @@ void hooksTests() {
     expectEquals(frameNumber, 2);
   });
 
+  test('_futureize does not leak sync uncaught exceptions into the zone', () async {
```

Comment:
If we're going through the trouble of maintaining an additional copy of `_futurize`, we should try to get more than one unit test out of the bargain. This test covers one of three error pathways, and there is also the success pathway, as examples.

Reply:
Thanks!